.travis.yml

```diff
@@ -49,30 +49,6 @@ matrix:
         - NUMPYSPEC="numpy"
         - TEST_GROUP_COUNT=2
         - TEST_GROUP=2
-    # Test SciPy with the debug version of Python.
-    # However travis only specifies regular or development builds of Python
-    # so we must install python3.6-dbg using apt.
-    - language: generic
-      name: "Python 3.7 Debug"
-      dist: bionic
-      env:
-        - USE_DEBUG=python3.7-dbg
-        - TESTMODE=fast
-        - NUMPYSPEC="--upgrade numpy"
-      addons:
-        apt:
-          packages:
-            - *common_packages
-            - python3.7-dbg
-            - python3.7-dev
-    - python: 3.7
```

Comment:
@ilayn , last thing before a merge. Did you mean to get rid of the `Power PC` entry?

Reply:
We may want to keep ppc64le entry in as a bookmark until/if we find another suitable location for that platform.

Reply:
oops, that was an overzealous delete. Sorry about that.